### DISCUSSION TITLE
Sou novo e preciso saber algo / I'm new and I need to know something

### DISCUSSION THREAD

**Discussion by Friskljo** [skyline-emu/skyline#948] [General]
Como faço para baixar os jogos? / How do I download games?

**Comment by firionm** [comment]


**Reply by ghost** [reply]
That's piracy

**Comment by ghost** [comment]
You don't download games you dump them from your switch

**Comment by alberto2098kl** [comment]
> ¿Cómo faço para baixar os jogos? / ¿Cómo descargo juegos?

Busca en Google tonto🤷🏻‍♂️
